### ISSUE TITLE
2.4.3 beta fails to build under gcc

### ISSUE THREAD

**Issue by kellogs** [tomojitakasu/RTKLIB#141]
git checkout rtklib_2.4.3
...
LANG=C ./makeall.sh 

```
% pos2kml/gcc
cc -c -Wall -O3 -ansi -pedantic -I../../../src -DTRACE ../pos2kml.c
cc -c -Wall -O3 -ansi -pedantic -I../../../src -DTRACE ../../../src/convkml.c
cc -c -Wall -O3 -ansi -pedantic -I../../../src -DTRACE ../../../src/solution.c
cc -c -Wall -O3 -ansi -pedantic -I../../../src -DTRACE ../../../src/geoid.c
cc -c -Wall -O3 -ansi -pedantic -I../../../src -DTRACE ../../../src/rtkcmn.c
cc -c -Wall -O3 -ansi -pedantic -I../../../src -DTRACE ../../../src/preceph.c
cc   pos2kml.o convkml.o solution.o geoid.o rtkcmn.o preceph.o  -lm -lrt -o pos2kml

% str2str/gcc
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../str2str.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/stream.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rtkcmn.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/solution.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/sbas.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/geoid.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcvraw.c
../../../src/rcvraw.c: In function 'decode_gal_inav':
../../../src/rcvraw.c:104:19: warning: unused variable 'time2' [-Wunused-variable]
     gtime_t time1,time2;
                   ^
../../../src/rcvraw.c:104:13: warning: unused variable 'time1' [-Wunused-variable]
     gtime_t time1,time2;
             ^
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/novatel.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/ublox.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/ss2.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/crescent.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/skytraq.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/gw10.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/javad.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/nvs.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/binex.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rcv/rt17.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rtcm.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rtcm2.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rtcm3.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/rtcm3e.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/preceph.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DENAGLO -DENAGAL -DENAQZS -DENACMP -DTRACE -DNFREQ=3 -DNEXOBS=3  -g ../../../src/streamsvr.c
cc   str2str.o stream.o rtkcmn.o solution.o sbas.o geoid.o rcvraw.o novatel.o ublox.o ss2.o crescent.o skytraq.o gw10.o javad.o nvs.o binex.o rt17.o rtcm.o rtcm2.o rtcm3.o rtcm3e.o preceph.o streamsvr.o  -lm -lrt -lpthread -o str2str
rcvraw.o: In function `input_raw':
/home/anon/rtklib/rtklib/RTKLIB/app/str2str/gcc/../../../src/rcvraw.c:961: undefined reference to `input_sbf'
rcvraw.o: In function `input_rawf':
/home/anon/rtklib/rtklib/RTKLIB/app/str2str/gcc/../../../src/rcvraw.c:989: undefined reference to `input_sbff'
collect2: error: ld returned 1 exit status
<builtin>: recipe for target 'str2str' failed
make: *** [str2str] Error 1

% rnx2rtkp/gcc
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../rnx2rtkp.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rtkcmn.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rinex.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rtkpos.c
../../../src/rtkpos.c: In function 'relpos':
../../../src/rtkpos.c:1515:12: warning: assuming signed overflow does not occur when assuming that (X + c) >= X is always true [-Wstrict-overflow]
 static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
            ^
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/postpos.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/solution.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/lambda.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/geoid.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/sbas.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/preceph.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/pntpos.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/ephemeris.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/options.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/ppp.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/ppp_ar.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rtcm.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rtcm2.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rtcm3.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/rtcm3e.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/ionex.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DNFREQ=3 -g ../../../src/qzslex.c
cc   rnx2rtkp.o rtkcmn.o rinex.o rtkpos.o postpos.o solution.o lambda.o geoid.o sbas.o preceph.o pntpos.o ephemeris.o options.o ppp.o ppp_ar.o rtcm.o rtcm2.o rtcm3.o rtcm3e.o ionex.o qzslex.o  -lm -lrt -o rnx2rtkp
rtkpos.o: In function `zdres':
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/rtkpos.c:890: undefined reference to `tidedisp'
postpos.o: In function `readpreceph':
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/postpos.c:574: undefined reference to `pppcorr_read'
ppp.o: In function `model_trop':
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/ppp.c:976: undefined reference to `pppcorr_trop'
ppp.o: In function `const_corr':
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/ppp.c:1047: undefined reference to `pppcorr_stec'
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/ppp.c:1035: undefined reference to `pppcorr_trop'
ppp.o: In function `model_iono':
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/ppp.c:1014: undefined reference to `pppcorr_stec'
ppp.o: In function `pppos':
/home/anon/rtklib/rtklib/RTKLIB/app/rnx2rtkp/gcc/../../../src/ppp.c:1310: undefined reference to `tidedisp'
collect2: error: ld returned 1 exit status
<builtin>: recipe for target 'rnx2rtkp' failed
make: *** [rnx2rtkp] Error 1

% convbin/gcc
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../convbin.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rtkcmn.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rinex.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/sbas.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/preceph.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcvraw.c
../../../src/rcvraw.c: In function 'decode_gal_inav':
../../../src/rcvraw.c:104:19: warning: unused variable 'time2' [-Wunused-variable]
     gtime_t time1,time2;
                   ^
../../../src/rcvraw.c:104:13: warning: unused variable 'time1' [-Wunused-variable]
     gtime_t time1,time2;
             ^
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/convrnx.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rtcm.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rtcm2.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rtcm3.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rtcm3e.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/pntpos.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/ephemeris.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/ionex.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/novatel.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/ss2.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/ublox.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/crescent.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/skytraq.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/gw10.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/javad.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/nvs.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/binex.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/rcv/rt17.c
cc -c -O3 -ansi -pedantic -Wall -Wno-unused-but-set-variable -I../../../src -DTRACE -DENAGLO -DENAQZS -DENAGAL -DENACMP -DNFREQ=6 -DNEXOBS=3 -g ../../../src/qzslex.c
cc   convbin.o rtkcmn.o rinex.o sbas.o preceph.o rcvraw.o convrnx.o rtcm.o rtcm2.o rtcm3.o rtcm3e.o pntpos.o ephemeris.o ionex.o novatel.o ss2.o ublox.o crescent.o skytraq.o gw10.o javad.o nvs.o binex.o rt17.o qzslex.o  -lm -lrt -o convbin
rcvraw.o: In function `input_raw':
/home/anon/rtklib/rtklib/RTKLIB/app/convbin/gcc/../../../src/rcvraw.c:961: undefined reference to `input_sbf'
rcvraw.o: In function `input_rawf':
/home/anon/rtklib/rtklib/RTKLIB/app/convbin/gcc/../../../src/rcvraw.c:989: undefined reference to `input_sbff'
collect2: error: ld returned 1 exit status
<builtin>: recipe for target 'convbin' failed
make: *** [convbin] Error 1

% rtkrcv/gcc
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../rtkrcv.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../vt.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtkcmn.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtksvr.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtkpos.c
../../../src/rtkpos.c: In function 'relpos':
../../../src/rtkpos.c:1515:12: warning: assuming signed overflow does not occur when assuming that (X + c) >= X is always true [-Wstrict-overflow]
 static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
            ^
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/geoid.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/solution.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/lambda.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/sbas.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/stream.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcvraw.c
../../../src/rcvraw.c: In function 'decode_gal_inav':
../../../src/rcvraw.c:104:19: warning: unused variable 'time2' [-Wunused-variable]
     gtime_t time1,time2;
                   ^
../../../src/rcvraw.c:104:13: warning: unused variable 'time1' [-Wunused-variable]
     gtime_t time1,time2;
             ^
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtcm.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/preceph.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/options.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/pntpos.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/ppp.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/ppp_ar.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/novatel.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/ublox.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/ss2.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/crescent.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/skytraq.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/gw10.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/javad.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/nvs.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/binex.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rcv/rt17.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/ephemeris.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rinex.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/ionex.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtcm2.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtcm3.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/rtcm3e.c
cc -c -Wall -O3 -ansi -pedantic -Wno-unused-but-set-variable -I../../../src -I.. -DTRACE -DENAGLO -DENAQZS -DENACMP -DNFREQ=3 -g ../../../src/qzslex.c
cc   rtkrcv.o vt.o rtkcmn.o rtksvr.o rtkpos.o geoid.o solution.o lambda.o sbas.o stream.o rcvraw.o rtcm.o preceph.o options.o pntpos.o ppp.o ppp_ar.o novatel.o ublox.o ss2.o crescent.o skytraq.o gw10.o javad.o nvs.o binex.o rt17.o ephemeris.o rinex.o ionex.o rtcm2.o rtcm3.o rtcm3e.o qzslex.o  -lm -lrt -lpthread -o rtkrcv
rtkpos.o: In function `zdres':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/rtkpos.c:890: undefined reference to `tidedisp'
rcvraw.o: In function `input_raw':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/rcvraw.c:961: undefined reference to `input_sbf'
rcvraw.o: In function `input_rawf':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/rcvraw.c:989: undefined reference to `input_sbff'
ppp.o: In function `model_trop':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/ppp.c:976: undefined reference to `pppcorr_trop'
ppp.o: In function `const_corr':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/ppp.c:1047: undefined reference to `pppcorr_stec'
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/ppp.c:1035: undefined reference to `pppcorr_trop'
ppp.o: In function `model_iono':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/ppp.c:1014: undefined reference to `pppcorr_stec'
ppp.o: In function `pppos':
/home/anon/rtklib/rtklib/RTKLIB/app/rtkrcv/gcc/../../../src/ppp.c:1310: undefined reference to `tidedisp'
collect2: error: ld returned 1 exit status
<builtin>: recipe for target 'rtkrcv' failed
make: *** [rtkrcv] Error 1

```

**Comment by tomojitakasu** [comment]
Sorry, no support for beta. It's just a working and development branch.
For stable, use 2.4.2 p11.

**Comment by tomojitakasu** [comment]
It is fixed in 2.4.3 b11.
